src/script.js

```diff
@@ -0,0 +1,79 @@
+// 폴더 공유하기 모달
+const shareModal = document.getElementById('shareFolderModal');
+const shareBtn = document.getElementById("shareFolderBtn");
+const shareSpan = document.querySelector("#shareFolderModal .close");
+
+shareBtn.onclick = function() {
+    shareModal.classList.add("show-modal");
+}
+
+shareSpan.onclick = function() {
+    shareModal.classList.remove("show-modal");
+}
+
+// 폴더 추가하기 모달
+const addModal = document.getElementById('addFolderModal');
+const addBtn = document.getElementById("addFolderBtn");
+const addSpan = document.querySelector("#addFolderModal .close");
+
+addBtn.onclick = function() {
+    addModal.classList.add("show-modal");
+}
+
+addSpan.onclick = function() {
+    addModal.classList.remove("show-modal");
+}
```

Comment:
share~, add~, to~ 모달 처리가 다 중복이라서 함수 하나로 만들어서 중복 줄여도 좋을 거 같아요!

```suggestion
// 폴더 추가하기 모달

function handleModalToggle(modalId, openBtnId) {
   const modal = document.getElementById(modalId);
   const openBtn = document.getElementById(openBtnId)
   const closeBtn = document.querySelector(`#${modalId} .close`)
   
   openBtn.onclick = function() {
    addModal.classList.add("show-modal");
   }

   closeBtn.onclick = function() {
    addModal.classList.remove("show-modal");
   }
   
   
}

handleModalToggle('shareFolderModal', "shareFolderBtn")
handleModalToggle('addFolderModal', "addFolderBtn")
handleModalToggle('toFolderModal', "toFolderBtn")

```

Reply:
아니면 모달로 만들것들을 array 로 만들어서 하는 것도 괜찮을 거 같아요
(to 보단 다른 변수명이 나을 거 같아용.. addToFolder 라던가... )
```
const modalList = ['share', 'add', 'to']
modalList.forEach((modal) => handleModalToggle(`${modal}FolderModal`, `${modal}FolderBtn`))
```